app/test_main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+from unittest import mock
+import pytest
+
+from app.main import can_access_google_page
+
+
+class TestCanAccessGoogle:
+    @pytest.mark.parametrize(
+        "valid_url,connection,expected_result",
+        [
+            (True, True, "Accessible"),
+            (True, False, "Not accessible"),
+            (False, True, "Not accessible"),
+            (False, False, "Not accessible"),
+        ]
+    )
+    @mock.patch("app.main.valid_google_url")
+    @mock.patch("app.main.has_internet_connection")
+    def test_can_access(self,
+                        mocked_google_url: mock,
+                        mocked_internet_connection: mock,
+                        valid_url: bool,
+                        connection: bool,
+                        expected_result: str) -> None:
```

Comment:
Place each argument including `self` on a new line so that there would be no arguments on the same line with opening and closing parenthesis. It will help to improve the code style and readability